peripheral.joystick/addon.xml.in

```diff
@@ -11,10 +11,10 @@
     provides_buttonmaps="true"
     library_@PLATFORM@="@LIBRARY_FILENAME@"/>
   <extension point="xbmc.addon.metadata">
-    <summary lang="en">Kodi Joystick Library</summary>
-    <description lang="en">This library provides joystick drivers and button maps. Multiple joystick APIs are supported, including DirectX, XInput, SDL and the Linux Joystick API.</description>
+    <summary lang="en_GB">Kodi Joystick Library</summary>
+    <description lang="en_GB">This library provides joystick drivers and button maps. Multiple joystick APIs are supported, including DirectX, XInput, SDL and the Linux Joystick API.</description>
     <platform>@PLATFORM@</platform>
-    <noicon>true</noicon>
-    <nofanart>true</nofanart>
+    <assets>
+    </assets>
```

Comment:
Why was noicon and nofanart even added initially? Only repo generator is allowed to add those (if the add-on would be shipped in a repo)
I also see no reason to add an empty assets tag.

Reply:
It prevents Kodi from hammering the server for icons and fanart. By default, Kodi tries to retrieve icon and fanart from the repo.

Reply:
If it doesn't come from a repo, which requests does it send?
I know what it does in a repo, but that's why the repo generator adds those tags and not the user.

Reply:
I can't test Kodi on linux right now so I'm not sure. Should `<assets>` be removed?

Reply:
assets, noicon and nofanart should be removed

Reply:
done